index.html

```diff
@@ -11,211 +11,251 @@
+  </ul>
+  <div class="item4">
+    <a href="https://github.com/Maedea9/Project-mobile-first" class="button">
+      <i class="see-project"></i>See this project &#8594;</a>
+  </div>
+</section>
+
+<div class="rectangle21v">
+  <img src="images/Rectangle 21 (3).png">
+</div>
+
+<section class="works5">
+
+  <h3>Project 6</h3>
+
+  <ul>
+    <li class="item1">Html/Css</li>
+    <li class="item2">Ruby on rails</li>
+    <li class="item3">JavaScript</li>
+  </ul>
+  <div class="item4">
+    <a href="https://github.com/Maedea9/Project-mobile-first" class="button">
+      <i class="see-project"></i>See this project &#8594;</a>
+  </div>
+</section>
+
+<section class="aboutme">
+  <h2>About</h2>
+</section>
+
+<section class="about2">
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+    aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod.</p>
+</section>
 
-    <section class="works5">
-      
-        <h3>Project 6</h3>
-     
-      <ul>
-        <li class="item1">Html/Css</li>
-        <li class="item2">Ruby on rails</li> 
-        <li class="item3">JavaScript</li>
-      </ul>
-        <div class="item4">
-            <a href="https://github.com/Maedea9/Project-mobile-first" class="button">
-                <i class="see-project"></i>See this project &#8594;</a>
+<div class="box">
+  <a href="#" class="button2">
+    <i class="getcv"></i>Get my CV &#8595;</a>
+</div>
+
+<section class="languages">
+  <div class="lang">Languages</div>
+  <ul>
+    <li class="lang1">&#x2022;Javascript (ES5/ES6)</li>
+    <li class="lang2">&#x2022;HTML</li>
+    <li class="lang3">&#x2022;CSS</li>
+  </ul>
+  <div class="figure1">
+    <img src="images/figure1.png">
+  </div>
+  </div>
+</section>
+
+<section class="frameworks">
+  <div class="framework-list">Frameworks</div>
+  <ul>
+    <li class="fram1">&#x2022;Bootstrap</li>
+    <li class="fram2">&#x2022;Ruby on rails</li>
+    <li class="fram3">&#x2022;RSpec</li>
+    <li class="fram4">&#x2022;Capibara</li>
+    <li class="fram5">&#x2022;Selenium</li>
+  </ul>
+  <div class="figure2">
+    <img src="images/ic_framework.png">
+  </div>
+  </div>
+</section>
+
+<section class="Skills">
+  <div class="skill1">Skills</div>
+  <ul>
+    <li class="skill2">&#x2022;Codekit</li>
+    <li class="skill3">&#x2022;Codepen</li>
+    <li class="skill4">&#x2022;Github</li>
+    <li class="skill5">&#x2022;GitLab</li>
+    <li class="skill6">&#x2022;Terminal</li>
+  </ul>
+  <div class="figure3">
+    <img src="images/ic_skills.png">
+  </div>
+</section>
+
+<section aria-label="Contact-me">
+    <div class="container-contact-me">
+      <h2 class="get-started">Get started</h2>
+      <p class="second-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+      <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
Nice job adding the `action` and `method` attributes to the form tag :+1: I suggest you fix the value added to the `action` attribute by replacing `{form_id}` with the correct `id` so that the messages get sent to your `Formspree` account.